index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
```

Comment:
Am I right that using `addonContext.registry` only looks up plugins within this addon, aka `in-repo-addon`s? Because if I'm using this in `ember-cli-htmlbars-inline-precompile' and register a `babel-plugin` it won't be picked up. Using `addonContext.app.registry` instead correctly picks up the plugins registered in other addons. @rwjblue can you confirm this?